src/backup/background/backend/google-drive/client.ts

```diff
@@ -174,6 +174,17 @@ export class GoogleDriveClient {
         return { id: response.id, created: true }
     }
 
+    // https://developers.google.com/drive/api/v3/reference/about
+    async getDriveStorageQuota() {
+        const response = await this._request(`/about?fields=storageQuota`, {
+            method: 'GET',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+        })
+        return response
```

Comment:
I realized the `/about` endpoint returns a whole lot more than just the storage quota (even though you're projecting everything else out with your query params). Given the method is named `getDriveStorageQuota`, it's better to make sure you pull out that object in here and just return that. Then when called, the caller doesn't have to do that, or worry about whatever else the object may have.